metricbeat/docs/modules/redis.asciidoc

```diff
@@ -33,8 +33,8 @@ redis://HOST[:PORT][?password=PASSWORD[&db=DATABASE]]
 [float]
 === Compatibility
 
-The Redis metricsets were tested with Redis 3.2.12, 4.0.11 and 5.0-rc4,  and are expected
-to work with all versions >= 3.0.
+The redis metricsets `info`, `key` and `keyspace` are compatible with the open source distribution of Redis. They were
```

Comment:
They should also work with non-OSS redis servers right?

Reply:
Thanks, fixed.